### PR TITLE
feat : lesson15

### PR DESCRIPTION
**Pull request by AyakaOnozuka131**
# [Issue No.15](https://github.com/kenmori/handsonFrontend/blob/master/work/markup/1.md#15)

**課題内容**
モーダル内に formをおいて、input(type number)値、input(type text)、を新たに作ったsubmitボタン押下で リクエスト、 APIから値を取ってきてください

## [CodeSandBox](https://codesandbox.io/s/lesson15-oppf0?file=/index.js) 
:Latest commit [53ccddc](https://github.com/AyakaOnozuka131/JavaScript-lessons/pull/15/commits/53ccddc68a513fd1f2291045b417f870941c3ed9)

## 実装上、不安なところ、重点的に見てもらいたいところ、聞きたいこと
- As for commenting out, please ignore it for my reminder.

### CODE REVIEW

File: lesson15/index.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    ul.appendChild(li).appendChild(loadingImg);
+};
+
+const showModal = () => {
+  modal.classList.add('isShow');
+}
+
+const hideModal = () => {
+  modal.classList.remove('isShow');
+}
```
Comment (ghost) [inline lesson15/index.js]: IMO
removeLoading()
removeButton()
It's easier to see if you align the writing style as well as this one.

```suggestion
const showModal = () => modal.classList.add("isShow");
const hideModal = () => modal.classList.remove("isShow");
```

File: lesson15/index.js [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+/**
+ * Async関数
+ * fetchData関数から値が返ってきた後、その値を返す
+ */
+const jsonUrl = 'https://myjson.dit.upm.es/api/bins/5tqv';
+const fetchListData = async () => {
+    createLoading();
+    try {
+        const inputValue = getInputValue();
```
Comment (ghost) [inline lesson15/index.js]: Q
 Why are you executing getInputValue in fetchListData?

File: lesson15/index.js (hunk 1 above)
Comment (AyakaOnozuka131) [reply lesson15/index.js]: Thank you for review!
Bracket omitted.
[304c5e2](https://github.com/AyakaOnozuka131/JavaScript-lessons/pull/15/commits/304c5e27ddec41a7aee51856274f0947d769fe90)

File: lesson15/index.js (hunk 2 above)
Comment (AyakaOnozuka131) [reply lesson15/index.js]: Thank you for pointing this out.
The getInputValue of fetchListData was inappropriate.
I set getInputValue as the argument of renderList.
Is this a problem?
[53ccddc](https://github.com/AyakaOnozuka131/JavaScript-lessons/pull/15/commits/53ccddc68a513fd1f2291045b417f870941c3ed9)

**Review by ghost — APPROVED** [review]

**Pull request merged**